src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6498,4 +6498,8 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
     <value>Methods attributed with 'UnmanagedCallersOnly' cannot have generic type parameters and cannot be declared in a generic type.</value>
     <comment>UnmanagedCallersOnly is not localizable.</comment>
   </data>
+  <data name="ERR_UnmanagedCallersOnlyMethodsCannotBeCalledDirectly" xml:space="preserve">
+    <value>'{0}' is attributed with 'UnmanagedCallersOnly' cannot be called directly. Obtain a function pointer to this method.</value>
```

Comment:
>cannot [](start = 59, length = 6)

"... and cannot be called directly"? Or perhaps "Method marked 'UnmanagedCallersOnly' cannot be called directly. ..." (It doesn't seem necessary to include the method name since the method name is the error span.)

Reply:
I originally didn't include the name in the error message, but I felt it looked better to have it, particularly for cases like GetEnumerator or Deconstruct methods that were attributed, as they don't appear in the error span.

---
In reply to: [482251935](https://github.com/dotnet/roslyn/pull/47311#discussion_r482251935) [](ancestors = 482251935)